src/scaffoldfitter/fitter.py

```diff
@@ -852,6 +853,37 @@ def getDataRMSAndMaximumProjectionErrorForGroup(self, groupName):
 
         return None, None
 
+    def getModelWorstElementJacobianInfo(self, mesh_group=None):
+        """
+        Get the information on the 3D element with the worst jacobian value (most negative).
+        Optional mesh group parameter allows the user to make the calculation over a different group
+        from the whole mesh.
+        :param mesh_group: Optional parameter to specify a particular mesh group to make the calculation over.
+        :return: Element identifier, minimum jacobian value. Values are -1, inf if there is no data or bad fields.
+        """
+        with ChangeManager(self._fieldmodule):
+            jacobian = calculate_jacobian(self._modelCoordinatesField)
+            report = report_on_lowest_value(jacobian, mesh_group if mesh_group else None)
+
+        return report
+
+    def getModelWorstElementJacobianInfoForGroup(self, group_name):
+        """
+        Get the information on the 3D element with the worst jacobian
+        value (most negative) of the 3D mesh group with the given name.
+        If the group_name is not a valid group name then None, None is returned.
+
+        :param group_name: Name of group to make calculation over.
+        :return: Element identifier, minimum jacobian value.
```

Comment:
Need to note returns -1, inf if failed to evaluate, but this is at odds with returning None, None below.